intlist.c

```diff
@@ -0,0 +1,269 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------
+// intlist.c
+// Program on main branch.
+// ------------------------------------------------------------------------------------------------
+
+
+#include <stdio.h>
+#include <malloc.h>
+#include "intlist.h"
+
+
+struct node {
+   int value;
+   struct node* link;
+};
+
+//Defines head pointer and fixes it to null
+struct node* Create () {
+   struct node* head;
+   printf ("\nAn empty linked list is created \n");

```

Comment:
The linked list implementation is a library function. It should not contain any print statements.